tools/initializer/src/main/java/org/sa/rainbow/initializer/Main.java

```diff
@@ -46,19 +46,14 @@ public static void main(String[] args) throws Exception {
                 templateSet = templateSetLoader.load();
             } else {
                 // Should not happen
+                System.out.println("Parse error: missing required option -t, please use -t <path_to_template>");
                 return;
             }
 
             Path configPath = null;
             // if user provides a config flag, parse the config path
             if (cmd.hasOption("c")) {
                 configPath = optionParser.handleConfigOption(cmd);
-            } else {
```

Comment:
It may not be a good idea to just delete the else block, we can print some hints instead.